daml/Marketplace/Clearing/Service.daml

```diff
@@ -170,7 +170,9 @@ template Service
         ctrl : Party
       controller ctrl
       do
-        void $ archive . fst <$> fetchByKey @MemberStanding (provider,customer)
+        (standingCid,_) <- fetchByKey @MemberStanding (provider,customer)
```

Comment:
This is ... much easier to understand! :smile:

Reply:
Weirdly enough, doing it the original way didn’t actually end up archiving the contract...which makes zero sense to me. So it’s possible I had changed something else between it working and it not working.